CryptographyCodingExerciseWeek13/CryptographyCodingExerciseWeek13/Assymetric.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Security.Permissions;
+using System.Security.Principal;
+using System.Text;
+namespace CryptographyCodingExerciseWeek13
+{
+    /// <summary>
+    /// Class that encrypts a string via Assymetric Encryption.
+    /// </summary>
+    public class Assymetric
+    {
+        /// <summary>
+        /// Method that takes a string parameter and encrypts the parameter using assymetric encryption.
+        /// </summary>
+        /// <param name="someText">String variable that gets passed in to be encrypted.</param>
+        /// <returns>String variable, decryptedValue is returned for evaluation.</returns>
+        public static string AssymetricEncrypt(string someText)
```

Comment:
Your symmetric and asymmetric methods do both encryption and decryption in the same method call. While this is ok for demonstrating the usage of each, it does not really make sense for a singe method to do both. In a real world implementation, you would want one method that encrypts a value and another method that decrypts an encrypted string. The same is true for digital signing - you'd want one method to sign some data and another to verify the signature. This would be important since you encrypt and sign data so you can transmit or share it safely in some way. But, if the encrypted or signed data never leaves the method, you can't share it or store it for later retrieval. (-0.2)